packages/components/switch/src/Switch.tsx

```diff
@@ -6,12 +8,45 @@ import { Label } from './SwitchLabel'
 export type SwitchProps = InputProps
 
 export const Switch = forwardRef<HTMLButtonElement, SwitchProps>(
-  ({ value = 'on', size = 'md', children, className, ...rest }, ref) => {
+  ({ value = 'on', size = 'md', children, className, id, disabled, ...rest }, ref) => {
+    const { id: controlledInputId, labelId: controlledLabelId } = useFormFieldState()
```

Comment:
The prefix controlled could be confusing I think we should use the word `field` somehow. If you don't destructure the object I think it would be better. For example:
```jsx
const field = useFormFieldState()
console.log(field.id)
```